### ISSUE TITLE
Will be better if images on the git page also include code link

### ISSUE THREAD

**Issue by deJQK** [rougier/scientific-visualization-book#19]
It might be better if the images on the project page are accompanied with the code link in the git (as in the book).

**Comment by rougier** [comment]
Good idea. Would like to make a PR? This might requires making thumbnails and re-organize the README.

**Comment by deJQK** [comment]
Thanks a lot. Sorry I am not very familiar with git or md so might not be able to help. Sorry for this.

**Comment by rougier** [comment]
No problem. I anyone want to propose a PR, that would be welcome (I do not have much time right now)
